ortools/linear_solver/BUILD.bazel

```diff
@@ -208,16 +197,14 @@ cc_library(
     }) + select({
         ":use_cplex": ["cplex_interface.cc"],
         "//conditions:default": [],
-    }) + select({
-        ":use_xpress": ["xpress_interface.cc"],
-        "//conditions:default": [],
     }),
     hdrs = [
         "linear_expr.h",
         "linear_solver.h",
         "linear_solver_callback.h",
         "scip_callback.h",
         "scip_helper_macros.h",
+        "MPSWriteError.h",
```

Comment:
no exception please.

Reply:
and catch them in the Write() implementation.
We can change the Write() method to return a bool. And we can log the error message.

Reply:
OK, no problem. We will work on it.